pravega-k8-log-collection/pravega-k8s-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+mkdir -p /tmp/logdir
+output_dir="/tmp/logdir"
+kubectl get po,svc,pvc --all-namespaces > $output_dir/kubectl-get-po-svc-pvc-all-namespace_output
+IFS=$'\n'
+for i in $(cat $output_dir/kubectl-get-po-svc-pvc-all-namespace_output | grep -i pravega)
```

Comment:
If the above command is updated, replace `grep -i pravega` with `grep -v NAME` to filter out headers.

Reply:
This depend on o/p of command suggested. Let me check

Reply:
Based on keeping `kubectl get po,svc,pvc --all-namespaces` haven't change anything in this space.